src/js/eventHandler.spec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { onFormSubmit, onEnterPress } from './eventHandlers';
```

Comment:
Just for keeping the naming convention consistent, I'd probably rename this file to `eventHandlers.spec.js`.

Reply:
Whoops! I didn't notice it was missing the 's', just changed it!